.eslintrc.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
       },
     ],
     "no-var": "error",
-    "prettier/prettier": "error",
+    "prettier/prettier": "error"
```

Comment:
NIT: Please add back the comma here. It's part of the styling convention that we have. Let's try and keep it consistent as much as possible.